testsuite/MDAnalysisTests/test_auxiliary.py

```diff
@@ -0,0 +1,108 @@
+
+from numpy.testing import (assert_equal, assert_raises, assert_almost_equal,
+                           assert_array_almost_equal)
+import MDAnalysis as mda
+
+from MDAnalysisTests.datafiles import AUX_XVG
+
+class BaseAuxReference(object):
+    def __init__(self):
+        self.n_steps = 5
```

Comment:
Probably best obtained from the data (`len(self.all_times)` for instance). It would avoid having `n_steps` out of sync with the data.